src/internal/utils/content-header-utils.ts

```diff
@@ -2,5 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 import { getGlobalFlag } from './global-flags';
 
-export const shouldRemoveHighContrastHeader = !!getGlobalFlag('removeHighContrastHeader');
-export const contentHeaderClassName = shouldRemoveHighContrastHeader ? '' : 'awsui-context-content-header';
+export const shouldRemoveHighContrastHeader = (): boolean => !!getGlobalFlag('removeHighContrastHeader');
+export const getContentHeaderClassName = (): string =>
```

Comment:
These are redundant. For simple expressions, auto-detection of return types works fine